src/other/index.jl

```diff
@@ -294,6 +294,25 @@ end
 @inline Base.getindex(x::AbstractIndex, rx::Regex) =
     getindex(x, filter(name -> occursin(rx, String(name)), _names(x)))
 
+# Levenshtein Distance
```

Comment:
please add a note about the source

Reply:
You can find pretty identical pseudocode all over the internet (e.g. https://en.wikipedia.org/wiki/Levenshtein_distance#Iterative_with_full_matrix) but will do

Reply:
Thank you. This is relevant for making sure that the whole code of DataFrames.jl can be distributed under MIT license.